app/services/flowcommerce_spree/import_items_hs_codes.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module FlowcommerceSpree
+  # A service object to import the data for product variants belonging to a flow.io Experience
+  class ImportItemsHsCodes
```

Comment:
Why do we need a separate task for this? Couldn't the HS code import just be added to the ImportExperienceItems service?

Reply:
Hmm unfortunately the information related to classifications is not included in the items information pulled from that endpoint. We need to call a different endpoint to retrieve the HS Codes, and this endpoint does not depend on the zone's. This is why I ended up separating into a different one. Do you think it would be best to make it more "general".. probably something like `ImportItemsAdditionalData` ?

Reply:
Hm, aren't they storing the HS Codes in the products' attributes? I remember they've told us tu put the HS codes into attributes when we first were trying to send them the HS Codes from Fulfil. 

If not in the attributes, then this separate service certainly makes sense.

Reply:
yep! I checked that information and unfortunately it was not there :(. After asking Jeff/Dave about it they mentioned that the classification information is not included within the data returned with the Product and we have to use this different Endpoint unfortunately. 

Probably something changed from last time on their side. The good news is that they will be working on getting a new endpoint to retrieve HS6 codes, which we will be able to use in the near future.